tsuru/client/auth.go

```diff
@@ -197,65 +213,65 @@ func (t *TeamUpdate) Flags() *gnuflag.FlagSet {
 		desc := "New team name."
 		t.fs.StringVar(&t.newName, "name", "", desc)
 		t.fs.StringVar(&t.newName, "n", "", desc)
+		t.fs.Var(&t.tags, "tag", "Team tag.")
```

Comment:
I think it would be best to add the full description for the tag here instead of adding it to `cmd.Info.Desc`. We'll render the help correctly.